public/javascripts/SVLabel/src/SVLabel/Keyboard.js

```diff
@@ -1,10 +1,53 @@
+window.addEventListener("keydown", function(e) {
+    // space and arrow keys
+    if([32, 37, 38, 39, 40].indexOf(e.keyCode) > -1) {
+        e.preventDefault();
+    }
+}, false);
+
+ // taken from 
+ // http://stackoverflow.com/questions/7799722/how-do-i-implement-move-forward-action-for-google-street-view
+ function difference(link, angle) {
```

Comment:
1. I don't think you need to have a separate function for this... You could, but it looks very simple...
2. The function name is not very meaningful. What do you mean by difference?
3. I prefer not to have functions in global scope. Put both difference() and movePano in the Keyboard module so they are private.